abis/GithubHint.json

```diff
@@ -1,2517 +1,83 @@
-  "source": "//! Github Hinting contract.\n//!\n//! Copyright 2016 Gavin Wood, Parity Technologies Ltd.\n//!\n//! Licensed under the Apache License, Version 2.0 (the \"License\");\n//! you may not use this file except in compliance with the License.\n//! You may obtain a copy of the License at\n//!\n//!     http://www.apache.org/licenses/LICENSE-2.0\n//!\n//! Unless required by applicable law or agreed to in writing, software\n//! distributed under the License is distributed on an \"AS IS\" BASIS,\n//! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n//! See the License for the specific language governing permissions and\n//! limitations under the License.\n\npragma solidity ^0.4.6;\n\n\ncontract GithubHint {\n\tstruct Entry {\n\t\tstring accountSlashRepo;\n\t\tbytes20 commit;\n\t\taddress owner;\n\t}\n\n\tmodifier whenEditAllowed(bytes32 _content) {\n\t\tif (entries[_content].owner != 0 && entries[_content].owner != msg.sender)\n\t\t\treturn;\n\t\t_;\n\t}\n\n\tfunction hint(bytes32 _content, string _accountSlashRepo, bytes20 _commit) whenEditAllowed(_content) public {\n\t\tentries[_content] = Entry(_accountSlashRepo, _commit, msg.sender);\n\t}\n\n\tfunction hintURL(bytes32 _content, string _url) whenEditAllowed(_content) public {\n\t\tentries[_content] = Entry(_url, 0, msg.sender);\n\t}\n\n\tfunction unhint(bytes32 _content) whenEditAllowed(_content) public {\n\t\tdelete entries[_content];\n\t}\n\n\tmapping (bytes32 => Entry) public entries;\n}\n",
-  "sourcePath": "/home/sorpaas/repos/github-hint/contracts/GithubHint.sol",
-  "ast": {
-    "absolutePath": "/home/sorpaas/repos/github-hint/contracts/GithubHint.sol",
-    "exportedSymbols": {
-      "GithubHint": [
-        95
-      ]
-    },
-    "id": 96,
-    "nodeType": "SourceUnit",
-    "nodes": [
+[
```

Comment:
Ooops! Looks like I generated ABI files in the wrong way.

Reply:
`cat build/contracts/GithubHint.json | jq .abi > abis/GithubHint.json`